bindings/python/README.rst

```diff
@@ -5,4 +5,47 @@ PyMongoArrow
        between MongoDB and Apache Arrow. See
        `GitHub <https://github.com/mongodb-labs/mongo-arrow/tree/main/bindings/python>`_
        for the latest source.
+:Documentation: Available at `mongo-arrow.readthedocs.io <https://mongo-arrow.readthedocs.io/en/latest/>`_.
 :Author: Prashant Mital
+
+**PyMongoArrow** is a companion library to PyMongo that contains tools
+for loading MongoDB query result sets as Apache Arrow tables, Pandas
+DataFrames or NumPy arrays.::
+
+  >>> from pymongoarrow.monkey import patch_all
```

Comment:
This doesn't get syntax highlighted like `requests` README because they use GitHub-flavored markdown

Reply:
According to https://github.com/github/markup/issues/914#issuecomment-387823670 we can use `.. code-block:: python`.

Reply:
Doesn't work AFAICT 
![image](https://user-images.githubusercontent.com/5883388/116161906-840c1400-a6a9-11eb-808c-4a3600f1b801.png)

Reply:
The syntax is like this:
```
- DataFrames or NumPy arrays.::
+ DataFrames or NumPy arrays.
+
+ .. code-block:: python
+
   >>> from pymongoarrow.monkey import patch_all
```